### PR TITLE
feat: Developers page [DEV-3766]

### PR DESCRIPTION
**Pull request by benyam7**

### CODE REVIEW

File: src/hooks.server.ts [hunk 1]
```diff
@@ -15,6 +16,7 @@ const authenticationHandler: Handle = async ({ event, resolve }) => {
 	switch (pathname) {
 		case '/home':
 		case '/billing':
```
Comment (lampkin-diet) [inline src/hooks.server.ts:18]: Should here be a `break` in case of picking the `billing` page?

Comment (benyam7) [reply src/hooks.server.ts:18]: @lampkin-diet No, we're guarding the client routes. In essence, it ensures that only authenticated users can access certain paths (`/home`, `/billing`, `/developers`) and redirects unauthenticated users back to the homepage if they try to access these paths.

File: src/routes/(app)/developers/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+		// 	'dslajsd lsdajlsdj fsaldjf alsdjf asldjflsjd flasdjlfjaslkdjfksjaldkjfldsajlfj daksdjf',
+		// 	() => {
+		// 		modalStore.clear();
+		// 		API_KEY_STEPS.EXISTING_API_KEYS = true;
+		// 		API_KEY_STEPS.CREATING_API_KEY = false;
+		// 		API_KEY_STEPS.NO_API_KEYS_YET = false;
+		// 	}
+		// );
+		// modalStore.trigger(dialog);
+	});
```
Comment (ankurdotb) [inline src/routes/(app)/developers/+page.svelte:59]: Necessary?

File: src/routes/(app)/developers/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+	</div>
+
+	<div class="flex w-full my-4 justify-end h-[1px] bg-tertiary-200"></div>
+
+	{#if API_KEY_STEPS.CREATING_API_KEY}
+		<div class="flex flex-col space-y-8">
+			<div class="flex w-full flex-col gap-1">
+				<span class="text-black text-xl">Name</span>
+				<span class="text-tertiary-800 text-xs md:text-sm"
+					>Give you api key a descriptive name which is easy to remember</span
```
Comment (ankurdotb) [inline src/routes/(app)/developers/+page.svelte:182]: Spellcheck/styling :) "GIve **your API** key..."

File: src/routes/(app)/developers/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+					class="py-2 p-2 w-full md:w-3/5 border border-tertiary-200 rounded-sm mt-4"
+					bind:value={apiKeyName}
+				/>
+			</div>
+
+			<div class="flex w-full flex-col gap-1">
+				<span class="text-black text-xl">Expires on</span>
+				<span class="text-tertiary-800 text-xs md:text-sm w-full md:w-3/5"
+					>Select a time duration until when the API will be active and usable. It is best to rotate
+					your kes often, by default this key will be valid for a year. You can choose to mak e this
```
Comment (ankurdotb) [inline src/routes/(app)/developers/+page.svelte:196]: spelling: "keys" not "kes"

**Review by ankurdotb — COMMENTED** [review]
1. Needs copyediting to make sure the text doesn't have spelling mistakes and styling errors.
2. I'm broadly fine with this being merged since it's targeting another PR branch, but please make sure they get resolved in the PR target branch.

Also on the PR target branch, would love to see another review from @guacamole since I haven't seen a review from her on this PR branch.

File: src/routes/(app)/developers/+page.svelte (hunk 2 above)
Comment (benyam7) [reply src/routes/(app)/developers/+page.svelte:59]: not really, accidentally got committed.

**Pull request merged**